draft-ietf-scitt-architecture.md

```diff
@@ -150,45 +150,6 @@ The core components are based on the Concise Signing and Encryption standard spe
 
 {::boilerplate bcp14-tagged}
 
-# Use Cases
```

Comment:
@hannestschofenig We should do two approaches, rather than blanket removal:

Option 1) My preferred Option: Somewhere in the Architecture text we refer to some known use cases and link it to the new document. Completely removing use case reference is not a good idea.

Option 2) Retain the title # Use Cases and on the next line refer to the reference to the new document

Reply:
A dedicated section here that will refer to the use cases document is required before merge.
Please note: the use-case subject of churn and has to be tracked independently!

Reply:
Add a link to https://datatracker.ietf.org/doc/draft-ietf-scitt-software-use-cases/

Reply:
:+1: to option 2

Reply:
Yep option 2. Then we can get back to editing the use case document :-)